gammapy/irf/tests/test_background.py

```diff
@@ -144,6 +144,15 @@ def test_background_3D_read():
     assert data.unit == "s-1 MeV-1 sr-1"
 
 
+@requires_data()
+def test_background_3D_read_gadf():
+    filename = "$GAMMAPY_DATA/tests/bkg_3d_full_example.fits"
+    bkg = Background3D.read(filename)
+    data = bkg.data.data
+    assert data.shape == (20, 15, 15)
```

Comment:
Just to be more explicit you could add an assert one the axes names as well...

Reply:
Done